### ISSUE TITLE
Net weight calculations

### ISSUE THREAD

**Issue by Priyanka-IAS** [frappe/erpnext#12414]
A new calculation for Net weight per item has been added to most of the documents in the new version. I see an issue with that:

- If the ‘quantity’ of the item is already weight based (as is the case in most of our documents), then the net weight is weight X weight which is a huge WRONG number. I have had to hide this number from all my documents and put in Net weight manually in a custom field due to this.

As Umair suggested, a possible resolution for this would be 

"Perhaps we will need to specify in UoM if it is countable or weighted. Based on that, we should or should-not do the Net Weight calculation"

**Comment by vishdha** [comment]
You can mention weight in Item Master.

**Comment by nabinhait** [comment]
May be you can set "Weight per unit" as 1 if stock UOM is weight based (For example, Kg).

**Comment by Priyanka-IAS** [comment]
Hi Nabin,

No, that won’t work out for us as we actually use the “Weight per unit” field in our case with real values. We use it to calculate the number of units which is also mentioned on our documents even though the main quantity is weight based.

We had actually requested for this field to be added as a custom field when we started up with ERPNext a few years ago. I didn’t how to do it myself then :)

Regards,

Priyanka
